tutorials/detect.ipynb

```diff
@@ -87,6 +114,40 @@
     ")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Adaptive smoothing\n",
```

Comment:
Maybe this specific question deserves a small notebook on its own. Something like how to produce an adaptively smoothed map? 
We probably need to re-discuss how to present how-to's that do not explicitly belong to a specific tutorial.